recipe/meta.yaml

```diff
@@ -35,10 +35,11 @@ test:
 
 about:
   home: https://github.com/tqdm/tqdm
-  license: Mozilla Public License 2.0 (MPL 2.0) or MIT License
+  license: MPLv2.0, MIT
   summary: 'A Fast, Extensible Progress Meter'
 
 extra:
   recipe-maintainers:
+    - casperdcl
```

Comment:
I can, but that would go against the general conda-forge policy as I understand it. No problem with just adding each member one at a time or in groups though, and @lrq3000 seems fine with it. Go ahead and add him too, and I'll merge once the CIs finish.